website/views.py

```diff
@@ -142,26 +143,29 @@ def index():
 
     user_id = get_current_user_id()
     if user_id:  # Logged in: return either landing page or user home page
-        all_institutions = (
-            Institution.objects.filter(
-                is_deleted=False,
-                nodes__is_public=True,
-                nodes__is_deleted=False,
-                nodes__type='osf.node'
+        if Switch.objects.get(name='ember_dashboard').is_active():
```

Comment:
Drive-by: you can do `waffle.switch_is_active('ember_dashboard')` here.

Reply:
is that better?